apps/web/src/modules/create-dao/components/AllocationForm/AllocationForm.tsx

```diff
@@ -125,7 +119,7 @@ export const AllocationForm: React.FC<AllocationFormProps> = ({ title }) => {
     setActiveSection(activeSection + 1)
   }
 
-  if (!signerAddress) return null
+  if (!address) return null
```

Comment:
In the rare case that address is null, do we really want the component to return null?

Reply:
this should be fixed when we rework the create dao flow. keeping as is for now to minimize changes in this upgrade